packages/cap-config-writer/test/fixture/cap-valid-cds-plugin-ui/package.json

```diff
@@ -1,9 +1,9 @@
 {
     "dependencies": {
-        "@sap/cds": "^6.8.2"
+        "@sap/cds": "^7.9.2"
```

Comment:
This is a major version bump in CDS, should the cap-config-writer also be a major bump?

Reply:
You are right, i will stick with ^6.8.2
cds-plugin-ui5: "0.9.3" is what i really need